SoundBuffered.js

```diff
@@ -60,6 +60,8 @@ SoundBuffered.prototype._createAudioNodes = function () {
 
 //▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
 SoundBuffered.prototype._destroyAudioNodes = function () {
+	if (!this._audioNodeReady) return;
```

Comment:
this fix the issue we have when `AudioManager` tries to unload the same sound more than one time.